app/src/main/java/com/tpp/theperiodpurse/ui/setting/ResetDatabase.kt

```diff
@@ -54,8 +55,10 @@ fun ResetDatabase(
         }
 
         LaunchedEffect(Unit) {
-            navController.popBackStack()
-            outController.navigate(OnboardingScreen.Welcome.name)
+            outController.navigate(OnboardingScreen.Welcome.name,
+                NavOptions.Builder()
+                    .setPopUpTo(OnboardingScreen.Welcome.name, true)
+                    .build())
```

Comment:
Can you try replacing this with just `outController.navigate(OnboardingScreen.Welcome.name)`? For me that worked